### DISCUSSION TITLE
slideshow packages are not work after build

### DISCUSSION THREAD

**Discussion by arishojaei** [nuxt/nuxt#9675] [Questions (Nuxt 2)]
I test [swiper](https://swiperjs.com/demos) and [splide](https://splidejs.com/) because I want to have a simple slideshow but both of them not working after build!

Currently, I want to use **splide** as a plugin in my nuxt project but I get this error after build.
![Screen_Shot_2021-08-13_at_21 10 11](https://user-images.githubusercontent.com/4179565/129395896-3c789d47-60c8-4d47-b452-01636b64c500.png)

I even didn't find any slideshow package for nuxtjs!

**Comment by kissu** [comment]
This one works for me: https://github.com/surmon-china/vue-awesome-swiper

Setup this as a plugin or import it per component. Working great. 👍🏻

**Reply by kissu** [reply]
Any news @arishojaei?
